src/legacy/ui/public/agg_types/controls/ip_range_type.tsx

```diff
@@ -29,22 +29,36 @@ enum IpRangeTypes {
 }
 
 function IpRangeTypeParamEditor({ value, setValue }: AggParamEditorProps<IpRangeTypes>) {
-  const useFromToLabel = i18n.translate('common.ui.aggTypes.ipRanges.useFromToButtonLabel', {
-    defaultMessage: 'Use From/To',
-  });
-  const useCidrMasksLabel = i18n.translate('common.ui.aggTypes.ipRanges.useCidrMasksButtonLabel', {
-    defaultMessage: 'Use CIDR Masks',
-  });
+  const options = [
+    {
+      id: IpRangeTypes.FROM_TO,
```

Comment:
I would make these id's more specific as they just end up as "mask" and "fromTo" which could conflict somewhere else if someone uses such broad terms.

Then you'll want to change the id reference from the `aria-labelledby` props of the inputs to these id's.

Reply:
Please have a look at the latest changes (I'm not sure that I got idea with aria-labelling right)